lib/money/money.rb

```diff
@@ -28,6 +30,7 @@ class UndefinedSmallestDenomination < StandardError; end
   def cents
     fractional
   end
+  alias_method :pence, :cents
```

Comment:
I'm fine with all changes but this. `cents` is a vestige from an early age of the gem that I'd like to remove for the more general `fractional`.